src/org/jgroups/stack/GossipRouter.java

```diff
@@ -428,7 +427,7 @@ public String[] supportedKeys() {
     }
 
     protected ByteArrayDataOutputStream getOutputStream(Address mbr, int size) {
-        return output_streams.computeIfAbsent(mbr, addr -> new ByteArrayDataOutputStream(size));
+        return output_streams.computeIfAbsent(mbr, __ -> new ByteArrayDataOutputStream(size));
```

Comment:
Whoa, fun, is this a new convention? Strictly speaking, this would violate the classical naming convention https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Fine by me! same as `ignored` instead of `addr`

Reply:
bad habit of mine when I don't need the variable in lambdas.

Reply:
I usually go for ignored, IIUC it's also picked up by IDEs and don't underline as 'unused'.